packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -190,13 +190,19 @@ const RealAccountSignup = ({
             title: WizardHeading,
         },
         {
-            body: local_props => (
-                <SignupErrorContent
-                    message={local_props.state_value.error_message}
-                    code={local_props.state_value.error_code}
-                    onConfirm={onErrorConfirm}
-                />
-            ),
+            body: local_props => {
+                let client_error_message = null;
+                if (local_props.state_value.error_code.message_to_client) {
+                    client_error_message = local_props.state_value.error_code.message_to_client;
+                }
+                return (
+                    <SignupErrorContent
+                        message={local_props.state_value.error_message || client_error_message}
+                        code={local_props.state_value.error_code}
+                        onConfirm={onErrorConfirm}
+                    />
+                );
+            },
```

Comment:
I suggest the following shorter solution with message_to_client taking priority over error_message. Please check if it works for you  🙂:
```suggestion
            body: local_props => {
                return (
                    <SignupErrorContent
                        message={
                            local_props.state_value.error_code.message_to_client || local_props.state_value.error_message
                        }
                        code={local_props.state_value.error_code}
                        onConfirm={onErrorConfirm}
                    />
                );
            },
```